### DISCUSSION TITLE
Ability to show a set of pages

### DISCUSSION THREAD

**Discussion by kksilas** [hrynko/vue-pdf-embed#245] [Ideas]
Hi there and thanks for the work you are putting into this.

Just my 2 cents, it would be nice if we could have the ability to pass an array of page numbers to show instead of an integer.
for example [1,2]

:)

**Comment by hrynko** [comment]
Hi @kksilas,

Thanks for your suggestion! I appreciate the idea, it's something I'll consider for future updates.
